client.js

```diff
@@ -1,41 +1,78 @@
-const WebSocketClient = require("ws");
+const WebSocket = require("ws");
 require("dotenv").config();
 const axios = require("axios").default;
 const fs = require("fs");
 
 //Sending HTTP requests with Axios is as simple as giving an object to the axios() function that contains all of the configuration options and data
 //TODO:set intervals, now it waits and then logs messages like crazy instead having a wait time in between every message
 
-const { APPID_INTAKE, APPID_OUTPUT, APPID_DLQ, URL, TIMEOUT, TOKEN, FILE } =
-  process.env;
+const {
+  APPID_INTAKE,
+  URL,
+  TIMEOUT,
+  FILE,
+  CLIENT_TOKEN,
+  OUTPUT_TOKEN,
+  DLQ_TOKEN,
+  INTAKE_TOKEN,
+} = process.env;
 
 async function main() {
```

Comment:
This main func should no longer be needed with the inclusion of websockets, as it will hold the connection open until it dies.

Reply:
async function main() is deleted.